packages/data/addon/mirage/routes/bard-lite-parsers.ts

```diff
@@ -43,34 +51,35 @@ export function parseFilters(filtersParam?: string) {
   return filterStrings.map(parseSingleFilter);
 }
 
-type Having = { [key: string]: { operator: string; values: string[] } };
+export type HavingOperator = 'gt' | 'gte' | 'lt' | 'lte' | 'eq' | 'neq' | 'bet' | 'nbet';
```

Comment:
Would this be more useful in the adapter?

Reply:
It's only used in fili mirage, but we could export it from there. I don't think we'll get much use from putting it in the adapter though